From 8199ee43c59eb6ca0408721c905e2681e38da763 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Fri, 23 Jul 2010 22:57:57 +0000
Subject: [PATCH] tests: Ignore the Test::Harness .prove file
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

We document how to run prove with the --state option in t/README. This
produces a .prove YAML file in the current directory. Change the t/
gitignore to ignore it, and clean it up on `make clean'.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/.gitignore |    1 +
 t/Makefile   |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/t/.gitignore b/t/.gitignore
index 7dcbb23..4e731dc 100644
--- a/t/.gitignore
+++ b/t/.gitignore
@@ -1,2 +1,3 @@
 /trash directory*
 /test-results
+/.prove
diff --git a/t/Makefile b/t/Makefile
index cf5f9e2..819b936 100644
--- a/t/Makefile
+++ b/t/Makefile
@@ -30,6 +30,7 @@ clean:
 	$(RM) -r 'trash directory'.* test-results
 	$(RM) t????/cvsroot/CVSROOT/?*
 	$(RM) -r valgrind/bin
+	$(RM) .prove
 
 aggregate-results-and-cleanup: $(T)
 	$(MAKE) aggregate-results
-- 
1.7.0.4

